package.json

```diff
@@ -19,70 +19,70 @@
     "lib": "lib"
   },
   "dependencies": {
-    "async": "2.1.4",
+    "async": "2.4.1",
     "big.js": "3.1.3",
-    "bluebird": "^3.4.7",
+    "bluebird": "^3.5.0",
     "boost-geospatial-index": "^1.0.3",
-    "bufferutil": "^2.0.0",
-    "bunyan": "^1.8.5",
-    "cli-color": "^1.1.0",
+    "bufferutil": "^3.0.1",
+    "bunyan": "^1.8.10",
+    "cli-color": "^1.2.0",
     "commander": "2.9.0",
-    "compare-versions": "^3.0.0",
-    "debug": "^2.6.0",
+    "compare-versions": "^3.0.1",
+    "debug": "^2.6.6",
     "denque": "^1.1.1",
     "dumpme": "^1.0.1",
     "easy-circular-list": "^1.0.13",
-    "elasticsearch": "13.0.1",
+    "elasticsearch": "13.1.1",
     "espresso-logic-minimizer": "^2.0.1",
-    "eventemitter2": "^3.0.0",
-    "fs-extra": "^1.0.0",
-    "glob": "^7.1.1",
-    "highwayhash": "^2.1.0",
-    "ioredis": "^2.5.0",
+    "eventemitter2": "^4.1.0",
+    "fs-extra": "^3.0.1",
+    "glob": "^7.1.2",
+    "highwayhash": "^2.1.1",
+    "ioredis": "^3.1.1",
     "js-combinatorics": "^0.5.2",
     "json-stable-stringify": "1.0.1",
     "json2yaml": "^1.1.0",
-    "jsonwebtoken": "^7.2.1",
+    "jsonwebtoken": "^7.4.1",
     "kuzzle-common-objects": "3.0.4",
     "lodash": "4.17.4",
-    "moment": "2.17.1",
-    "ms": "^1.0.0",
+    "moment": "2.18.1",
+    "ms": "^2.0.0",
     "ngeohash": "0.6.0",
-    "node-interval-tree": "^1.0.0",
+    "node-interval-tree": "^1.1.4",
     "node-units": "0.1.7",
-    "node-uuid": "1.4.7",
+    "node-uuid": "1.4.8",
     "passport": "0.3.2",
     "pm2": "2.4.6",
-    "proper-lockfile": "^2.0.0",
-    "rc": "1.1.6",
-    "readline-sync": "^1.4.6",
-    "reify": "^0.4.7",
-    "rimraf": "^2.5.4",
+    "proper-lockfile": "^2.0.1",
+    "rc": "1.2.1",
+    "readline-sync": "^1.4.7",
+    "reify": "^0.11.24",
+    "rimraf": "^2.6.1",
     "sorted-array": "^2.0.1",
-    "utf-8-validate": "^3.0.1",
-    "validator": "^6.2.1",
-    "ws": "2.3.1"
+    "utf-8-validate": "^3.0.2",
+    "validator": "^7.0.0",
+    "ws": "3.0.0"
```

Comment:
ws@3 introduces a [breaking change](https://github.com/websockets/ws/releases/tag/3.0.0) in not storing the `upgradeReq` object in the socket anymore.
Not that critical here as we are only using it in debug & log message but still...